lisp/mastodon-tl.el

```diff
@@ -399,21 +399,35 @@ Move forward (down) the timeline unless BACKWARD is non-nil."
   (goto-char (point-max))
   (mastodon-tl--property 'toot-id t))
 
+(defun mastodon-tl--as-string(id)
```

Comment:
`mastodon-tl--id-as-string`?

Reply:
This could be used anywhere we wish to convert a number to a string. To generalize I will rename the variable `ID` to `NUMERIC`.

Reply:
ok.